Framework/CurveFitting/src/FuncMinimizers/DampedGaussNewtonMinimizer.cpp

```diff
@@ -30,6 +30,7 @@ DECLARE_FUNCMINIMIZER(DampedGaussNewtonMinimizer, Damped GaussNewton)
 DampedGaussNewtonMinimizer::DampedGaussNewtonMinimizer(double relTol)
     : IFuncMinimizer(), m_relTol(relTol) {
   declareProperty("Damping", 0.0, "The damping parameter.");
+  declareProperty("Debug", false, "Turn on the debug output.");
```

Comment:
Debug logging level is for debugging the code. This flag is for the user to observe what `Fit` is doing internally and maybe to help to figure out why a fit went wrong. I don't think the user would want to be bombarded by all the irrelevant information from the debug level.